site/_posts/2017-02-20-februarys-final-weeks-work.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Last Week in ManageIQ - February's Final Week's Work"
+author: AllenBW
+date: 2017-02-21
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Woah first last week of February for 2017, this is :taco::dancer: **huge**. *Almost* as :taco::dancer: huge as the LQIMIQ low down prepared for you by your friendly neighbourhood developer [Allen](https://github.com/allenbw)!  So without further ado lets :woman_juggling:! 
+
+## Featured
+So here are the raw numbers:
+* ManageIQ Core with [94 PRs merged][manageiq PRs merged] merged (awarded most overachieving!) :1st_place_medal: 
+* ManageIQ Classic had [50 PRs][manageiq-ui-classic PRs merged] merged (eerily similar number as last week, for those of you keeing track, awarded most copycat!) :2nd_place_medal: 
+* ManageIQ Content saw [9 PRs][manageiq-content PRs merged] merged (awarded most generic! no really, checkout those prs!) :3rd_place_medal: 
+* ManageIQ Service UI saw [9 PRs][manageiq-ui-service PRs merged] merged (awarded most housekeeping!) :4th_place_medal:
+* Amazon provider had [4 PRs][manageiq-providers-amazon PRs merged] merged (awarded most resilient!) :5th_place_medal:
+* Azure provider had [3 PRs][manageiq-providers-azure PRs merged] merged (awarded most resilient!) :6 th_place_medal:
```

Comment:
@AllenBW I thought that might be the case :) How about the space in :6 th_place_medal: , is that intentional as well?